src/TextBoxGroup/TextBoxGroup.tsx

```diff
@@ -88,7 +99,7 @@ export const TextBoxGroup: React.FunctionComponent<TextBoxGroupProps> = (props:
                         </div>
                     }
                 </div>
-                <div className="alert alert-danger">{props.error}</div>
+                {showErrorMessage && !props.success ? <div className="alert alert-danger">{props.error}</div> : null}
```

Comment:
same here for moving the props.success to the showErrorMessage logic in the use effect

Reply:
fixed in e90bd8f73c3e2216a90ccb30713b1707f1241063